src/main/java/frc/robot/commands/ClimbEnable.java

```diff
@@ -27,7 +27,7 @@ public void initialize() {
     m_subsystem.climbEnable();
     m_subsystem.resetTargetedHeight();
 
-    if (m_subsystem.getclimbingenable() == true) {
+    if (m_subsystem.getclimbingmode() == true) {
```

Comment:
'Enabled' is actually the better naming scheme for documentation. 'Mode' is ambiguous about what the true/false values mean. Consider something like 'isClimbingEnabled' instead.

'== true' is unnecessary as well

Reply:
climbing enable isn't something I deleted, I was confused as to what climbing mode meant